src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,78 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+type PropsPagination = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}: PropsPagination) => {
+  const totalPages = Math.ceil(getNumbers(1, total).length / perPage);
```

Comment:
The use of `getNumbers(1, total).length` to calculate `totalPages` seems to be an unnecessary overhead. You are generating an array of numbers up to `total` just to get its length, which is equivalent to `total` itself. Consider simplifying this calculation by using `total` directly.